app/views/hyrax/dashboard/sidebar/_tasks.html.erb

```diff
@@ -1,7 +1,7 @@
   <% if can? :review, :submissions %>
     <li class="h5"><%= t('hyrax.admin.sidebar.tasks') %></li>
     <%= menu.nav_link(hyrax.admin_workflows_path) do %>
-      <span class="fa fa-flag"></span> <span class="sidebar-action-text"><%= t('hyrax.admin.sidebar.workflow_review') %></span>
+      <span class="fa fa-flag" aria-hidden="true"></span> <span class="sidebar-action-text"><%= t('hyrax.admin.sidebar.workflow_review') %></span>
     <% end %>
   <% end %>
 
```

Comment:
Manage Users isn't actually shown as far as I know, but the icon fa-user should probably also have aria-hidden as well.